events/events_settings.py

```diff
@@ -0,0 +1,5 @@
+REST_FRAMEWORK = {
```

Comment:
I would just put this in the project root settings.py. No need to have an extra import. When the application is deployed in production, the settings file is completely overridden.

Reply:
If you wanted to make the application "re-useable" add a README inside the events app and write down the required SETTINGS